README.md

```diff
@@ -2,6 +2,7 @@
 
 Demo of koa.
 
+[![Codacy Badge](https://api.codacy.com/project/badge/Grade/9f602a8d2d2a4ba3bb9bab0695247ac6)](https://app.codacy.com/app/jjangga0214/koa-helloworld?utm_source=github.com&utm_medium=referral&utm_content=jjangga0214/koa-helloworld&utm_campaign=Badge_Grade_Dashboard)
```

Comment:
There is naming inconsistency as well. 
Other badges start with a capital letter and use underscore.

Possible discussion topic: should we set consistency?